pynxtools_xps/vms/vamas.py

```diff
@@ -89,18 +89,18 @@
 
 UNITS: dict = {
     "beam_xray/excitation_energy": "eV",
-    "beam_xray/extent": "micro-m",
+    "beam_xray/extent": "um",
     "beam_xray/particle_charge": "C",
-    "beam_xray/source_beam_width_x": "micro-m",
-    "beam_xray/source_beam_width_y": "micro-m",
+    "beam_xray/source_beam_width_x": "um",
+    "beam_xray/source_beam_width_y": "um",
```

Comment:
Is it `u` or `micro` if it is micro, you will get error in nomad as well. you can try: `micrometer`